frontend/herd.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file This file contains the Herd function. This function fetches herd for a
+ *       given `herdId` (parsed from the url), as well as the individuals
+ *       belonging to that herd.
+ */
+import React from 'react'
+import {Link, useParams} from "react-router-dom";
+
+import { get } from './communication';
+
+/**
+ * Shows herd information, with a list of all individuals belonging to that
+ * herd.
+ */
+export function Herd() {
+  let { herdId } = useParams();
+  const [herd, setHerd] = React.useState(undefined)
+
+  React.useLayoutEffect(() => {
+    get(`/api/herd/${herdId}`).then(
+      data => setHerd(data.herd),
+      error => console.error(error)
+    )
+  }, [])
+
+  return <>
+    {herd
+      ? <>
+          <h2>{herd.name ?? `Besättning ${herd.id}`}</h2>
+          <ul>
+            {herd.individuals.map(individual => {
+              return <Link key={individual.id} to={`/individual/${individual.id}`}><li>{individual.name}</li></Link>
+            })}
+          </ul>
+        </>
+      : ''}
```

Comment:
You can do `{herd && <> ... </>}` instead of `{herd ? <> ... </> : ''}`. False values are already not included in the dom (such as `undefined` here)